providers/jackson2/src/main/java/org/jboss/resteasy/plugins/providers/jackson/PatchMethodFilter.java

```diff
@@ -60,7 +60,7 @@ public void filter(ContainerRequestContext requestContext) throws IOException
          Object object;
          try
          {
-            object = methodInvoker.invokeDryRun(request, response);
+            object = methodInvoker.invokeDryRun(request, response).toCompletableFuture().getNow(null);
```

Comment:
@FroMage Wouldn't be better to wait some time and use `.get(some_reasonable_timeout)`? Same for all `getNow` usage in this PR. Is it possible, that default value (`null`) is returned, because data is not yet available?

Reply:
So, in fact this particular plugin just does not support async. It's almost never a good idea to wait on a timeout because that may block the very thread that would cause the async value to be emitted (on vertx for example).
I used `getNow` in all places where async wasn't supported.
Now, with the async filter support, it may be possible to make this particular filter async, but in general I'm wary of changing things unless there's user code to require this change.